src/formatting_stack/protocols/spec.clj

```diff
@@ -53,10 +53,10 @@ so that final users can locate them and configure them."
   (spec/keys :req-un [::filename
                       ::source
                       ::msg
-                      ::level
-                      ::column
-                      ::line]
-             :opt-un [::msg-extra-data
+                      ::level]
+             :opt-un [::column
```

Comment:
Since we're not covering with tests that a specific macro can cause the original issue (which honestly would be pretty contrived, if not unreliable), I'd add over here a comment explaining why these keys are (now) optional.

That comment would be our "tests" ;p

Also interesting in a future to make these keys required for linters that we know that can't fail. OTOH there's a risk of hitting similar issues again